pages/common/gifsicle.md

```diff
@@ -1,20 +1,32 @@
 # gifsicle
 
-> Create GIFs.
+> GIF manipulator.
 > More information: <https://www.lcdf.org/gifsicle>.
 
-- Optimise a GIF:
+- Optimize a GIF as a new file:
 
-`gifsicle --batch --optimize=3 {{amin.gif}}`
+`gifsicle {{path/to/input.gif}} --optimize=3 -o {{path/to/output.gif}}`
 
-- Make a GIF animation with gifsicle:
+- Unoptimize a GIF in place:
 
-`gifsicle --delay={{10}} --loop *.gif > {{anim.gif}}`
+`gifsicle -b {{path/to/input.gif}} --unoptimize`
 
-- Extract frames from an animation:
+- Extract a frame from an animation:
```

Comment:
```suggestion
- Extract a frame from a GIF:
```

Reply:
I think what the original author was implying was this stuff only matters for animated GIFs. But I agree with this. I'm gonna commit your suggestions and make another rework.